cylc/flow/time_parser.py

```diff
@@ -130,12 +130,12 @@ def __init__(
         )
 
         if isinstance(context_start_point, str):
-            context_start_point = self._get_point_from_expression(
-                context_start_point, None)[0]
+            context_start_point, _ = self._get_point_from_expression(
+                context_start_point, None)
```

Comment:
FYI: This is subjective, the way this was done before saved an unnecessary assignment.